pkg/ottl/contexts/ottlmetric/README.md

```diff
@@ -0,0 +1,32 @@
+
+## Paths
+In general, the Metric Context supports accessing pdata using the field names from the [metrics proto](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto).  All integers are returned and set via `int64`.  All doubles are returned and set via `float64`.
+
+The following fields are the exception.
+
+| path                                   | field accessed                                                                                                | type                                                                    |
+|----------------------------------------|---------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------|
+| resource                               | resource of the data point being processed                                                                    | pcommon.Resource                                                        |
+| resource.attributes                    | resource attributes of the data point being processed                                                         | pcommon.Map                                                             |
+| resource.attributes\[""\]              | the value of the resource attribute of the data point being processed                                         | string, bool, int64, float64, pcommon.Map, pcommon.Slice, []byte or nil |
+| instrumentation_scope                  | instrumentation scope of the data point being processed                                                       | pcommon.InstrumentationScope                                            |
+| instrumentation_scope.name             | name of the instrumentation scope of the data point being processed                                           | string                                                                  |
+| instrumentation_scope.version          | version of the instrumentation scope of the data point being processed                                        | string                                                                  |
+| instrumentation_scope.attributes       | instrumentation scope attributes of the data point being processed                                            | pcommon.Map                                                             |
+| instrumentation_scope.attributes\[""\] | the value of the instrumentation scope attribute of the data point being processed                            | string, bool, int64, float64, pcommon.Map, pcommon.Slice, []byte or nil |
+
+## Enums
+
+The Metric Context supports the enum names from the [metrics proto](https://github.com/open-telemetry/opentelemetry-collector/blob/main/pdata/pmetric/metrics.go#L129-L136).
```

Comment:
These lines don't match the enum definition when I try to access this. It may be better to have the link point to the file at a certain commit to pin the file contents.

Reply:
Since pdata is always changing and contrib is always kept up to date it'll be hard to link to the exact spot accurately and keep it up-to-date.  I updated it now to just go to the file, which should change less often. Also fixed the proto link.